packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -70,13 +70,27 @@
         <protected>false</protected>
         <shortDescription>Clear filter</shortDescription>
     </labels>
+    <labels>
+        <fullName>quantic_ClearFilterFacet</fullName>
+        <value>Clear filter for the {{0}} facet.</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Clear filter for the potato facet.</shortDescription>
+    </labels>
     <labels>
         <fullName>quantic_ClearFilter_plural</fullName>
         <value>Clear {{0}} filters</value>
         <language>en_US</language>
         <protected>false</protected>
         <shortDescription>Clear 2 filters</shortDescription>
     </labels>
+    <labels>
+        <fullName>quantic_ClearFilterFacet_plural</fullName>
```

Comment:
Could we simplify our life a little bit and just have a "clear all filters for the [value] facet" that always apply?